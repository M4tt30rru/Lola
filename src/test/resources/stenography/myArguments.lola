##Import "std.lola"
##Find ##Type(_type) ##Identifier(_name)(##ArgumentDeclarationList) { ##Any(_c1) (#) ##Any(_c2) }
 ##replace ##(_type) ##(_name)(##(','.join([_arg._type.name + ' ' + _arg._name.name for _arg in _args]))){##(_c1) (##(','.join(str(_arg._name.name) for _arg in _args))) ##(_c2)}

void main(Integer argc, String argv) {
  printf("Hello!");
  do_main_silently(#);
}
