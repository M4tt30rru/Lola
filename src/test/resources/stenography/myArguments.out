void main(Integer argc, String argv) {
  printf("Hello!");
  do_main_silently(argc, argv);
}
